xml/install_caasp_heat_templates.xml

```diff
@@ -116,7 +116,505 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
+   </mediaobject>
+  </informalfigure>
+  <para>
+   CaaSP Admin Node Install: Screen 6
+  </para>
+  <informalfigure>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="caasp_6.png" width="70%" format="PNG"></imagedata>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="caasp_6.png" width="90%" format="PNG"></imagedata>
+    </imageobject>
+   </mediaobject>
+  </informalfigure>
+  <para>
+   CaaSP Admin Node Install: Screen 7
+  </para>
+  <informalfigure>
+   <mediaobject>
+    <imageobject role="fo">
+     <imagedata fileref="caasp_7.png" width="70%" format="PNG"></imagedata>
+    </imageobject>
+    <imageobject role="html">
+     <imagedata fileref="caasp_7.png" width="90%" format="PNG"></imagedata>
+    </imageobject>
+   </mediaobject>
+  </informalfigure>
+ </section>
+ <section>
+  <title>Deploy CaaSP Stack Using &o_orch; CaaSP Playbook</title>
+  <procedure>
+   <step>
+    <para>
+     Install the <package>caasp-openstack-heat-templates</package> package on a
+     machine with &productname; repositories:
+    </para>
+    <screen>&prompt.ardana;zypper in caasp-openstack-heat-templates</screen>
+    <para>
+     The installed templates are located in
+     <filename>/usr/share/caasp-openstack-heat-templates</filename>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Run <filename>heat-caasp-deploy.yml</filename> on the &lcm; to create a
+     CaaSP cluster with &o_orch; templates from the
+     <literal>caasp-openstack-heat-templates</literal> package.
+    </para>
+    <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
+&prompt.ardana;ansible-playbook -i hosts/localhost heat-caasp-deploy.yml</screen>
+   </step>
+   <step>
+    <para>
+     In a browser, navigate to the &o_dash; UI to determine the floating IP
+     address assigned to the admin node.
+    </para>
+   </step>
+   <step>
+    <para>
+     Go to http://<replaceable>ADMIN-NODE-FLOATING-IP</replaceable>/ to bring
+     up the Velum dashboard.
+    </para>
+   </step>
+   <step>
+    <para>
+     Follow the steps for <literal>bootstrap</literal> and <literal>add nodes
+     to the cluster</literal> to bring up the CaaSP Kubernetes cluster.
+    </para>
+   </step>
+  </procedure>
+ </section>
+ <section>
+  <title>Deploy CaaSP Cluster with Multiple Masters Using &o_orch; CaasP
+  Playbook</title>
+  <procedure>
+   <step>
+    <para>
+     Install the <package>caasp-openstack-heat-templates</package> package on a
+     machine with &productname; repositories:
+    </para>
+    <screen>&prompt.ardana;zypper in caasp-openstack-heat-templates</screen>
+    <para>
+     The installed templates are located in
+     <filename>/usr/share/caasp-openstack-heat-templates</filename>.
+    </para>
+   </step>
+   <step>
+    <para>
+     On the &lcm;, run the <filename>heat-caasp-deploy.yml</filename> playbook
+     and pass parameters for caasp_stack_name, caasp_stack_yaml_file and
+     caasp_stack_env_yaml_file.
```

Comment:
<literal>caasp_stack_name</literal>, <literal>caasp_stack_yaml_file</literal> and <literal>caasp_stack_env_yaml_file</literal>.